stam.py

```diff
@@ -42,19 +42,6 @@ def get_stampy_modules():
     if not os.path.exists(database_path):
         raise Exception("Couldn't find the stampy database file at " + f"{database_path}")
 
-    if ENVIRONMENT_TYPE == "production":
```

Comment:
TJ and Aprillion confirmed that this can be deleted

Reply:
actually, I only confirmed the old sentience module was moved to git, so it shouldn't be needed
not 100% sure that @robertskmiles didn't misuse this mechanism to run some new secret code 😅 

but let's delete and if Rob needs some secret code directly on the server, we can call it something else than `sentience` (the joke was funny while we were a closed group, but now this is a public project and it might a bit too edgy if AI safety bot imported a secret module called sentience)